roles/nginxplus/files/conf/http/dss-staging.conf

```
@@ -44,6 +52,8 @@ server {
         proxy_pass http://dss-staging;
         proxy_set_header X-Forwarded-Host $host;
         proxy_set_header X-Forwarded-Proto https;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-Proto https;
```

I was looking at this PR to see if we can get it merged. As @tpendragon suggested, this line should be the X-Forwarded-For line. We already have X-Forwarded-Proto two lines above. 
```suggestion
        proxy_set_header X-Forwarded-For $remote_addr;
```